File: examples/docs-code-snippets/src/main/kotlin/examples/kotlin/recipes/RecipeWithIngredientNameOverrides.kt
package examples.kotlin.recipes

import com.ing.baker.recipe.kotlindsl.ExperimentalDsl
import com.ing.baker.recipe.kotlindsl.recipe
import examples.kotlin.interactions.ShipOrder

@ExperimentalDsl
object RecipeWithIngredientNameOverrides {
    val recipe = recipe("example") {
        interaction<ShipOrder> {
            ingredientNameOverrides {
                "orderId" to "orderNumber"
            }
        }
    }
}

File: examples/docs-code-snippets/src/main/scala/examples/scala/recipes/RecipeWithIngredientNameOverrides.scala
package examples.scala.recipes

import com.ing.baker.recipe.scaladsl.Recipe
import examples.scala.interactions.ShipOrder

object RecipeWithIngredientNameOverrides {
  val recipe: Recipe = Recipe("example")
    .withInteraction(
      ShipOrder.interaction
        .withOverriddenIngredientName("orderId", "orderNumber")
    )
}